### PR TITLE
Documentation for Medallia destinations connector

### PR DESCRIPTION
**Pull request by muqing-zhou**
Added for review by Adobe documentation team:

- New file "medallia-connector.md" for new destination connector with Adobe Experience Platform (FYI we also have a separate Launch extension in the same VOC folder with file "medallia.md")
- Supporting screenshots

### CODE REVIEW

File: help/destinations/catalog/voice/medallia-connector.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+<img src="/help/destinations/assets/catalog/voice/medallia-destination-oauth.png" width="60%">
+
+
+### Fill in destination details {#destination-details}
+To configure details for the destination, fill in the required fields and select **[!UICONTROL Next]**.
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Import API Name]**: Name of the Medallia Import API (also known as Web Feed) to be used in this connection. Youcan activate different segments to different Import APIs to trigger different survey programs
```
Comment (vgiurgiu) [inline help/destinations/catalog/voice/medallia-connector.md]: Minor comment: You can (missing space). And please add a period at the end of the sentence.

File: help/destinations/catalog/voice/medallia-connector.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+![Image of sample mapping](/help/destinations/assets/catalog/voice/medallia-destination-mapping.png)
+
+
+>[!IMPORTANT]
+> 
+> Share with your Medallia delivery team the exact **Attribute names** for every target custom attribute you map using **Target field** > **Select custom attributes** > **Attribute name**. You may wish to take a screenshot of the mapping page to share directly.
+
+
+## Exported data {#exported-data}
```
Comment (vgiurgiu) [inline help/destinations/catalog/voice/medallia-connector.md:120]: One comment here. The purpose of this section is for readers to verify that the data is indeed being correctly exported to your destination. 

So, for example, you could tell them to go into the Medallia interface and check that the profiles or segments out of Experience Platform are being populated.

File: help/destinations/catalog/voice/medallia-connector.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+Data is exported in batches of 1000 profiles or 30 minute increments (whichever occurs first) as profiles qualify for your activated segments.
```
Comment (vgiurgiu) [inline help/destinations/catalog/voice/medallia-connector.md]: Three should also be a Data Governance section in the self-service template. That is boilerplate information for any destination, so please keep it on the page.

**Review by vgiurgiu — COMMENTED** [review]
@muqing-zhou , great work! Thanks for submitting the documentation. I just left a few small comments. Please let me know if you have any questions.

File: help/destinations/catalog/voice/medallia-connector.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
+---
+title: Medallia connection
+description: Activate profiles for targeted Medallia surveys and feedback collection to better understand customer needs and expectations.
+---
+
+# Medallia connection
+
+## Overview {#overview}
```
Comment (vgiurgiu) [inline help/destinations/catalog/voice/medallia-connector.md:8]: Let's please leave a blank line between headings and the paragraph that follows them. This comment applies to all headings in the documentation page. Thanks!

File: help/destinations/catalog/voice/medallia-connector.md (hunk 1 above)
Comment (muqing-zhou) [reply help/destinations/catalog/voice/medallia-connector.md]: Thanks for catching! Updated

File: help/destinations/catalog/voice/medallia-connector.md (hunk 2 above)
Comment (muqing-zhou) [reply help/destinations/catalog/voice/medallia-connector.md:120]: Gotcha - updated the section - let me know what you think.

File: help/destinations/catalog/voice/medallia-connector.md (hunk 3 above)
Comment (muqing-zhou) [reply help/destinations/catalog/voice/medallia-connector.md]: Added to the end of the template

File: help/destinations/catalog/voice/medallia-connector.md (hunk 4 above)
Comment (muqing-zhou) [reply help/destinations/catalog/voice/medallia-connector.md:8]: Done!

**Comment by vgiurgiu** [comment]
@muqing-zhou  - thanks for making the updates. 

I don't have any other questions at this time. After our engineering team reviews the destination submission and confirms that everything works as expected, we'll be able to publish the documentation as well.

**Comment by vgiurgiu** [comment]
@muqing-zhou - we reviewed the destination submission one more time in our team and came across one small thing that I overlooked in my earlier review. 

As per the self-service template (https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en#overview ), can you please add a note similar to the one in the Overview section, that mentions that the documentation was authored by Medallia and indicates a support channel where your team can be reached? 

```
>[!IMPORTANT]
>
>This documentation page was created by the *YOURDESTINATION* team. For any inquiries or update requests, please contact them directly at *Insert link or email address where you can be reached for updates*
```

**Comment by muqing-zhou** [comment]
@vgiurgiu thanks for that callout! added the note to the overview section and our contact info

**Comment by vgiurgiu** [comment]
@muqing-zhou - We moved this pull request over into our enterprise Git instance and published the documentation page here: 
https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/voice/medallia-connector.html

Thank you for your work! Closing this PR.

**Pull request closed without merging**